plugins/haskell.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "haskell",
+  "version": "0.0.1",
+  "match": "^(ghc|haskell\\.compiler\\.(.*))$",
+  "readme": "Haskell plugin",
+  "packages": [
+    "path:{{ .Virtenv }}"
+  ],
+  "create_files": {
+    "{{ .Virtenv }}/flake.nix": "haskell/flake.nix"
```

Comment:
this map is also confusing me a bit: are there some docs on how to interpret the plugin format?

Reply:
I'll write some docs in follow up